CDS/js/test/TestMediaAtomLib.js

```diff
@@ -122,7 +122,7 @@ describe('mediaAtomLib', () => {
                         'X-Gu-Tools-Service-Name': 'content_delivery_system'
                     }})
                 .put(URI, {
-                    uri: 'youtube_url'
+                    uri: 'https://www.youtube.com/watch?v=youtube_id'
```

Comment:
would this not post the literal string "https://www.youtube.com/watch?v=youtube_id", rather than substituting in the uri as I assume is the case?